e2e/Microsoft.Azure.Devices.E2ETests/TwinE2ETests.cs

```diff
@@ -391,6 +391,74 @@ static public void ClassCleanup()
             await _Twin_DeviceSetsReportedPropertyAndServiceReceivesIt(Client.TransportType.Amqp_WebSocket_Only);
         }
 
+        [TestMethod]
+        [TestCategory("Twin-E2E")]
+        public async Task Twin_ServiceDoesNotCreateNullPropertyInCollection_Mqtt()
+        {
+            // GitHub Issue #243
+            await _Twin_ServiceDoesNotCreateNullPropertyInCollection(Client.TransportType.Mqtt_Tcp_Only);
+        }
+
+        [TestMethod]
+        [TestCategory("Twin-E2E")]
+        public async Task Twin_ServiceDoesNotCreateNullPropertyInCollection_MqttWs()
+        {
+            // GitHub Issue #243
+            await _Twin_ServiceDoesNotCreateNullPropertyInCollection(Client.TransportType.Mqtt_WebSocket_Only);
+        }
+
+        [TestMethod]
+        [TestCategory("Twin-E2E")]
+        public async Task Twin_ServiceDoesNotCreateNullPropertyInCollection_Amqp()
+        {
+            // GitHub Issue #243
+            await _Twin_ServiceDoesNotCreateNullPropertyInCollection(Client.TransportType.Amqp_Tcp_Only);
+        }
+
+#if NETCOREAPP2_0
+        // GitHub Issue #302
```

Comment:
Convention: we mark active issues with `// TODO: #nnn <explanation>`. In this case it would be `// TODO: #302 In NetCoreApp2.0 the test is failing with TimeoutException`.

Reply:
Fixed